### DISCUSSION TITLE
Roaster/artisan temp match

### DISCUSSION THREAD

**Discussion by elguapo1911** [artisan-roaster-scope/artisan#1082] [Q&A]
My Diedrich ir12 has a digital temp Guage that is about five degrees f off from what artisan says. Is there a way to make them match?

**Comment by poundy** [comment]
Sure. If you care. But you probably shouldn't. Different probes, in different places, measuring different temperatures is totally normal. 

Adjustment: see https://artisan-scope.org/docs/setup/ and read the "configure language, temps, and sampling" section on symbolic temperature formulae.
